packages/cli/yaml/validator/src/rules/valid-example-type/__test__/valid-example-type.test.ts

```diff
@@ -198,15 +198,13 @@ describe("valid-example-type", () => {
             {
                 severity: "error",
                 relativeFilepath: RelativeFilePath.of("object.yml"),
-                message:
-                    'Example is missing required property "foo". Object1 -> (extends) commons.ObjectWithFooAndBar -> foo',
```

Comment:
QQ: do we want to keep this second sentence? Could be nice, since it's pointing out the extended object the property is on

Reply:
Down to add it back if you think thats better